spec/models/setting_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Setting, type: :model do
+  let(:uploaded_file) { fixture_file_upload('profile_picture.jpg') }
+  let(:blob) { ActiveStorage::Blob.create_and_upload!(io: uploaded_file, filename: uploaded_file.original_filename) }
+  subject { Setting }
+
+  %i[onboarding_logo onboarding_hero].each do |getter|
+    setter = "#{getter}="
+    blob_id_getter = "#{getter}_blob_id"
+    blob_id_setter = "#{getter}_blob_id="
```

Comment:
Too much complexity for a test? :thinking:

Reply:
@phoeinx Can you review today? If not, maybe @tillprochaska ?

Reply:
Today I won't make it, but if Till doesn't find time today, I'll try tomorrow.

Reply:
> Too much complexity for a test? 🤔

Using loops in test cases always feels a bit like this and I guess it would be a little easier to follow if the expectations were simply duplicated for `onboarding_logo` and `onboarding_hero`, but I’m fine with the current solution as well.